unittests.py

```diff
@@ -0,0 +1,199 @@
+"""Unit tests to check the maths functions."""
+
+import logging
+import numpy as np
+import unittest
+
+import rotations
+import rigid_transformations as rigids
+import visualization
+
+
+class TestGeomstatsMethods(unittest.TestCase):
+
+    def test_regularize_rotation_vector(self):
+        rot_vec_1 = 2.5 * np.pi * np.array([0., 0., 1.])
+        rot_vec_1 = rotations.regularize_rotation_vector(rot_vec_1)
+        rot_vec_1_expected = np.pi / 2. * np.array([0., 0., 1.])
+        self.assertTrue(np.allclose(rot_vec_1, rot_vec_1_expected))
+
+        rot_vec_2 = 1.5 * np.pi * np.array([0., 1., 0.])
+        rot_vec_2 = rotations.regularize_rotation_vector(rot_vec_2)
+        rot_vec_2_expected = np.pi / 2. * np.array([0., -1., 0.])
+        self.assertTrue(np.allclose(rot_vec_2, rot_vec_2_expected))
+
+        rot_vec_3 = 11 * np.pi * np.array([1., 2., 3.])
+        rot_vec_3 = rotations.regularize_rotation_vector(rot_vec_3)
+        fact = 0.84176874548664671 * np.pi / np.sqrt(14)
+        rot_vec_3_expected = fact*np.array([-1., -2., -3.])
+        self.assertTrue(np.allclose(rot_vec_3, rot_vec_3_expected))
+
+        rot_vec_4 = 1e-15 * np.pi * np.array([1., 2., 3.])
+        rot_vec_4 = rotations.regularize_rotation_vector(rot_vec_4)
+        rot_vec_4_expected = rot_vec_4
+        self.assertTrue(np.allclose(rot_vec_4, rot_vec_4_expected))
+
+        rot_vec_5 = 1e-11 * np.array([12., 1., -81.])
+        rot_vec_5 = rotations.regularize_rotation_vector(rot_vec_5)
+        rot_vec_5_expected = rot_vec_5
+        self.assertTrue(np.allclose(rot_vec_5, rot_vec_5_expected))
+
+    def test_representations_of_rotations(self):
```

Comment:
you might want to split your tests a bit more. Like one test per assert, but that's just me being picky :)

Reply:
You mean per assert or per assert's type (corresponding to a method)?